packages/core/src/tools/web-fetch.ts

```diff
@@ -80,6 +81,9 @@ export class WebFetchTool extends BaseTool<WebFetchToolParams, ToolResult> {
         type: 'object',
       },
     );
+    if (config.getProxy()) {
```

Comment:
```suggestion
    const proxy = config.getProxy();
    if (proxy) {
```

And then can re-use below

Reply:
thanks for your advice, accept changes~